src/ports/catalog/queries.ts

```diff
@@ -637,9 +640,17 @@ const getNFTsWithOrdersCTE = (filters: CatalogQueryFilters) => {
       SQL`.order AS orders
         WHERE 
             orders.status = 'open' 
-            AND orders.expires_at_normalized > NOW() AND nft.owner_address = orders.owner`
+            AND orders.expires_at_normalized > NOW()`
     )
 
+    .append(
```

Comment:
Would you mind adding a comment here saying that in the newest case we're only interested in the newest items limited by the limit parameter?

Reply:
done!